### PR TITLE
feat(next): Hackathon error page

### PR DESCRIPTION
**Pull request by sardesam**
## Because

- We want to start building out pages in preparation for M3.

## This pull request

- Introduces a fully accessible desktop version for the error page.
- Hackathon demo.
- Includes starter and backlog tasks for error page.

## Issue that this pull request solves

Closes N/A (Hackathon demo)

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] I have added necessary documentation (if appropriate).

### CODE REVIEW

**Comment by sardesam** [comment]
@xlisachan @StaberindeZA the code includes comments that will be removed but the bonus content can be added here (unless there is another PR) before merging.

File: apps/payments/next/app/[offeringId]/error/page.tsx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+          Error confirming subscription…
+        </h1>
+        <div className="flex items-center justify-center gap-2 text-green-900 mb-4">
+          <CheckIcon />
+          <h2 className="font-semibold text-sm">30-day money-back guarantee</h2>
+        </div>
+      </header>
+
+      {/* body */}
+      <article className="component-card border-t-0 mb-6 pt-4rounded-t-lg text-grey-600 px-4 pb-12">
```
Comment (pdehaan) [inline apps/payments/next/app/[offeringId]/error/page.tsx:107]:
```suggestion
      <article className="component-card border-t-0 mb-6 pt-4 rounded-t-lg text-grey-600 px-4 pb-12">
```

**Review by StaberindeZA — APPROVED** [review]
r+. I'm not too worried about the comments. This page will get cleaned up when the error page gets picked up in milestone M3b

File: apps/payments/next/app/[offeringId]/error/page.tsx (hunk 1 above)
Comment (sardesam) [reply apps/payments/next/app/[offeringId]/error/page.tsx]: Good catch! Fixed.

**Pull request merged**